backend/application/syson-application/src/test/java/org/eclipse/syson/application/controllers/diagrams/general/view/GVEdgeCreationTests.java

```diff
@@ -88,7 +88,7 @@ public class GVEdgeCreationTests extends AbstractIntegrationTests {
     private EdgeCreationTester edgeCreationTester;
 
     @Autowired
-    private SemanticCheckerFactory semanticCheckerFactory;
+    private SemanticRunnableFactory semanticRunnableFactory;
```

Comment:
I renamed the `SemanticCheckerFactory` to `SemanticRunnableFactory` since it can be used to create any kind of runnable (not only checkers)